modules/configure-certificates-api-add-named.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientca-for-the-openshift-api-server_{context}"]
+= Configure an additional clientCA for the OpenShift API server
+
+. Import the CA certificate in a configmap of the openshift-config namespace. The CA file must be in PEM format.
++
+[source,terminal]
+----
+$ oc create configmap client-ca-custom -n openshift-config --from-file=ca-bundle.crt=ca.crt
+----
+
+. Patch the APIServer instance.
++
+[source,terminal]
+----
+$oc patch apiserver cluster --type=merge -p '{"spec": {"clientCA": {"name": "client-ca-custom"}}}'
+----
+
+. Import the CA certificate in a configmap of the openshift-config namespace. The CA file must be in PEM format.
+
+After adding the new CA, any API request providing an x.509 client certificate signed by the new CA and matching a valid user is successfully authenticated.
```

Comment:
Could this line be reworded as a description/intro for the procedure and moved up to follow the module title?

Reply:
rhn-support-palonsor[Pablo Alonso Rodriguez](https://issues.redhat.com/secure/ViewProfile.jspa?name=rhn-support-palonsor)(https://issues.redhat.com/secure/ViewProfile.jspa?name=pbertera%40redhat.com)
@stlaz 

Pablo, Pietro, Standa,

Please look this over from a QE and SME perspective, making sure I captured what you expected. 

Thanks!

Neal